### PR TITLE
Add empty declaration inspection

### PR DESCRIPTION
**Pull request by grenki**

### CODE REVIEW

File: .idea/inspectionProfiles/idea_default.xml [hunk 1]
```diff
@@ -55,7 +55,6 @@
     <inspection_tool class="AndroidLintLabelFor" enabled="false" level="WARNING" enabled_by_default="false" />
     <inspection_tool class="AndroidLintLibraryCustomView" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="AndroidLintLocalSuppress" enabled="false" level="ERROR" enabled_by_default="false" />
-    <inspection_tool class="AndroidLintMangledCRLF" enabled="false" level="ERROR" enabled_by_default="false" />
```
Comment (zolotov) [inline .idea/inspectionProfiles/idea_default.xml]: do not commit anything from `.idea/` directory

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -198,6 +198,10 @@
     <localInspection language="go" displayName="Unused parameter inspection" groupPath="Go"
                      groupName="Declaration redundancy" enabledByDefault="true" level="WARNING"
                      implementationClass="com.goide.inspections.unresolved.GoUnusedParameterInspection"/>
+
+    <localInspection language="go" displayName="Empty declaration inspection" groupPath="Go"
+                     groupName="Declaration redundancy" enabledByDefault="true" level="WARNING"
```
Comment (zolotov) [inline resources/META-INF/plugin.xml]: I think it should be `WEAK_WARNING`

File: tests/com/goide/inspections/GoEmptyDeclarationInspectionTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  protected boolean isWriteActionRequired() {
+    return false;
+  }
+
+  public void testConstVoidDeclaration() {
+    doTest(GoEmptyDeclarationInspection.QUICK_FIX_NAME, true);
+  }
+
+  public void testVarVoidDeclarationWithInnerComment() {
+    doTestNoFix("", true);
```
Comment (zolotov) [inline tests/com/goide/inspections/GoEmptyDeclarationInspectionTest.java]: `doTestNoFix` looks for quick fix named `""`. Looks like it's impossible to fail the test, use `GoEmptyDeclarationInspection.QUICK_FIX_NAME` here

File: resources/META-INF/plugin.xml (hunk 2 above)
Comment (grenki) [inline resources/META-INF/plugin.xml]: Sergey told me about WARNING instead WEAK WARNING

**Comment by zolotov** [comment]
Looks good to me. Please make fixes according to comments.

After that please investigate how `com.intellij.codeInspection.CleanupLocalInspectionTool` works. Looks like this inspection is a good candidate for `Cleanup` action

**Comment by grenki** [comment]
Create new pull request with corrections.

**Pull request closed without merging**